commitlint.config.js

```diff
@@ -6,27 +6,19 @@ module.exports = {
       'always',
       [
         'client',
-        'client/cordova',
-        'client/cordova/android',
-        'client/cordova/apple',
-        'client/cordova/apple/ios',
-        'client/cordova/apple/macos',
-        'client/electron',
-        'client/electron/linux',
-        'client/electron/windows',
+        'client/android',
+        'client/ios',
+        'client/linux',
+        'client/macos',
+        'client/windows',
         'devtools',
         'docs',
         'infrastructure',
-        'infrastructure/build',
         'manager',
         'manager/linux',
-        'manager/mac',
+        'manager/macos',
         'manager/windows',
-        'service',
-        'service/linux',
-        'service/windows',
-        'storybook',
-        'www',
+        'storybook'
```

Comment:
Can we remove storybook too? You're either working on the client, the manager, or if it's the storybook infra I'd argue that's a devtools thing?